src/main/resources/i18n/messages.properties

```diff
@@ -556,3 +556,7 @@ reviewScore.four=4
 reviewScore.three=3
 reviewScore.two=2
 reviewScore.one=1
+chat.replay=replay
+replay.copyLink=Copy link for chat
```

Comment:
I wouldn't call it "for chat" because hyperlinks aren't dedicated to being used in chats